src/Events/Editor/Full_Site/Templates.php

```diff
@@ -73,7 +73,7 @@ public function add_events_archive( array $query_result, $query, string $templat
 	 */
 	public function get_template_events_archive() {
 		$template_content = file_get_contents(
-			dirname( \Tribe__Main::instance()->plugin_path ) . '/src/Events/Editor/Full_Site/Templates/archive-events.html'
+			dirname( TRIBE_EVENTS_FILE ) . '/src/Events/Editor/Full_Site/Templates/archive-events.html'
```

Comment:
Why not use the plugin path? I would love to avoid using the main file constant outside of the base loading

Reply:
Because the plugin path is giving an incorrect path:
`dev/plugins/src/Events/Editor/Full_Site/Templates/archive-events.html`

Reply:
Ah, it's the `dirname` causing the issue 🤦🏼 
Tribe__Main::instance()->plugin_path . /src/Events/Editor/Full_Site/Templates/archive-events.html'
works correctly